sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1261,6 +1397,138 @@ no queue state is changed. This design is because the queues are already in
 the state the user desires, so if the function threw an exception in this case,
 the application would likely swallow it and then proceed.
 
+=== External events
+
+The default behavior is that the event associated with a graph node cannot be 
+used to synchronize workload outside the graph.
+To overcome this limitation, users can define and use graph exernal events.
+These events can be used as regular sycl events. Users can therefore use them
```

Comment:
In theory, these events can be used in the same way as regular events. So, yes, we can wait on these events.
As for profiling, it will require a specific use case in the implementation, but it's doable.

Reply:
What kind of usages for profiling do you think would be supported? I imagine the profiling information for an external event would need to represent the entire graph to that point as it may not be possible to query individual commands.

Reply:
Given that we haven't had a request for profiling for these events and the mentioned implementation burden I'm inclined to disallow profiling for now and make that change in the future as a separate piece of work if it's useful.